pywhy_stats/categorical.py

```diff
@@ -0,0 +1,293 @@
+"""Independence test among categorical variables using power-divergence tests.
```

Comment:
The module name `categorical.py` is quite general. I don't think this will cover all possible categorical tests. What about `power_divergence.py`? Maybe it also makes sense now to have sub-packages like "independence". Then we can have another subpackage like "ksample" etc., depending on the test.

Reply:
Sure